enderio-machines/src/main/java/crazypants/enderio/machines/machine/teleport/GuiTravelAccessable.java

```diff
@@ -29,11 +30,15 @@
   private static final int ID_PUBLIC = 0;
   private static final int ID_PRIVATE = 1;
   private static final int ID_PROTECTED = 2;
+  
+  //private static final int ID_CLOSE_WINDOW_BUTTON = 12615;
```

Comment:
Can we remove references to the close window button for now? More than likely it will need a full rewrite to implement, so this commented code may not be relevant ultimately.